Dockerfile

```diff
@@ -1,14 +1,21 @@
+FROM sbtscala/scala-sbt:eclipse-temurin-jammy-11.0.22_7_1.9.9_2.13.13 as builder
+WORKDIR /app
+COPY project project
+COPY src src
```

Comment:
since "build.sbt" is less likely to be edited than "src", for build cache purposes, it would make more sense to swap the order of these likes.

but it's a tiny optimization of the build time, feel free to merge without changing it